README.md

```diff
@@ -22,10 +22,40 @@ Available targets:
 
 ## How to use
 
-**NOTE:** The target project _must_ be added as a bindmount to the `/ci-harness` directory.
+This project can be utilized one of two ways, via docker or via a Makefile include.
 
-```bash
-IMAGE="plus3it/tardigrade-ci:latest"
-docker pull "$IMAGE"
-docker run --rm -ti -v "my-project-dir:/ci-harness/my-project" "$IMAGE" help
-```
+### Via Docker
+
+  **NOTE:** The target project _must_ be added as a bindmount to the `/ci-harness` directory.
+
+  ```bash
+  IMAGE="plus3it/tardigrade-ci:latest"
+  docker pull "$IMAGE"
+  docker run --rm -ti -v "my-project-dir:/ci-harness/my-project" "$IMAGE" helps
+  ```
+
+### Makefile Include
+
+1. Create a Dockerfile in the project in which you wish to utilize these ci tools with the following content.
+
+  **NOTE:** This Dockerfile is intended to be used to enable version pinning of the underlying toolset.
+
+  ```bash
+  FROM plus3it/tardigrade-ci:0.0.2
+
+  WORKDIR /ci-harness
+  ENTRYPOINT ["make"]
+  ```
+
+2. Add a simple makefile with these two lines
+
+  ```bash
+  SHELL := /bin/bash
+
+  -include $(shell curl -sSL -o .tardigrade-ci "https://raw.githubusercontent.com/plus3it/tardigrade-ci/master/bootstrap/Makefile.bootstrap"; echo .tardigrade-ci)
+
+  ```
+
+3. Add `*tardigrade-ci*` to your `.gitignore` file
```

Comment:
This should be just `.tardigrade-ci` and `tardigrade-ci`. Don't need the wildcards.

Reply:
updated